plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -285,6 +287,61 @@ public void testSelectivelyOptimizingLeavesEqualityDeletes()
         Assertions.assertThat(loadTable(tableName).currentSnapshot().summary().get("total-equality-deletes")).isEqualTo("1");
     }
 
+    @Test
+    public void testEqualityDeletesWithDifferentSchemas()
+            throws Exception
+    {
+        String tableName = "test_equality_delets_different_schemas_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM tpch.tiny.nation", 25);
+        Table icebergTable = updateTableToV2(tableName);
```

Comment:
unnecessary: the table has already format_version set to `2` by default.

Reply:
I refer to the writing of other tests, not sure if it is already version 2.

Reply:
Use `computeActual("show create table " + tableName);` to verify.
I'm creating a cleanup PR for this purpose.